common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -136,4 +136,10 @@ public Object generate() {
             }
         });
     }
+
+    public enum ModLoader {
+        Forge,
```

Comment:
Enum are still to be considered consts, i.e. FORGE, ... We have been very bad at enforcing this. I am surprised that Spotless does not catch this.

Reply:
Unless there have been a shift in Java conventions lately? That might have happened. But to me, the UpperCamelCase is strictly reserved for classes.

Reply:
This should be enforced better in general, but done